app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for i in phrase:
+        if i.lower() == letter.lower():
```

Comment:
please, use descriptive and correct variable names.

Reply:
I fixed solution.